language/de/common.php

```diff
@@ -319,7 +319,7 @@
 		2	=> '%d unsichtbare Mitglieder',
 	),
 	'HIDDEN_USERS_TOTAL'		=> array(
-		1	=> '%d unsichtbares',
+		1	=> '%d unsichtbarer',
```

Comment:
Danke für die Erklärung. Die Änderung hat also mit der Verbesserung "Ersetze Account mit Benutzerkonto" nichts zu tun. Vielmehr handelt es sich dabei um eine unabhängige Fehlerbereinigung. Prinzipiell ist es sinnvoll für unabhängige Änderungen unabhängige Pull Requests zu eröffnen. Ich nehme jedoch an, dass wir in diesem Fall den existierenden Pull Request gesamthaft akzeptieren können.